docs/src/examples/components/Toolbar/Content/ToolbarExampleMenu.shorthand.tsx

```diff
@@ -1,48 +1,39 @@
-import * as React from 'react'
+import { callbackLogFormatter } from '@stardust-ui/code-sandbox'
+import { useBooleanKnob, useLogKnob } from '@stardust-ui/docs-components'
 import { Toolbar } from '@stardust-ui/react'
+import * as React from 'react'
 
 const ToolbarExampleMenuShorthand = () => {
-  const [menuOpen, setMenuOpen] = React.useState(false)
+  const [menuOpen, setMenuOpen] = useBooleanKnob({ initialValue: false, name: 'menuOpen' })
```

Comment:
See my prev comment, this boolean knob is not working as expected.

Reply:
Reverted it for now, I don't that it's related to the knob itself. Should be tested separately.